ui/admin/tests/acceptance/targets/create-test.js

```diff
@@ -323,6 +325,33 @@ module('Acceptance | targets | create', function (hooks) {
     assert.strictEqual(getTCPTargetCount(), tcpTargetCount);
   });
 
+  test('can add aliases during target creation', async function (assert) {
```

Comment:
None blocking comment: Wondering, should we add another test case were we attempt to add  2 or 3 aliases? thoughts? Not necessary on this PR, since this is an initiative WIP 😉